File: dayOne/Python/NumberGuesser.py
import random

random_number = random.randint(1,100)


for count in range(1,6):
    user_input = int(input("Enter A Number from 1 to 100: "))
    if random_number == user_input:
        new_count = count
        if new_count == 1:
            print(f"The Correct Number is {user_input} With {new_count} attempts used and a Legendary rating")
        if new_count == 2:
            print(f"The Correct Number is {user_input} With {new_count} attempts used and a Excellent rating")
        if new_count == 3 or new_count == 4:
            print(f"The Correct Number is {user_input} With {new_count} attempts used and a Good rating")
        if new_count == 5:
            print(f"The Correct Number is {user_input} With {new_count} attempts used and a Close rating")
    
        break
    
    elif random_number != user_input:
        if user_input < random_number:
            print("Lower than Stake")
        elif user_input > random_number:
            print("Higher that Stake")
            
print()
print("Ohh! Chances ran out")   
print("Better Luck Next Time")            

File: dayOne/Java/NumberGuesser.java
    import java.util.Scanner;
    
    import java.util.Random;
    
    public class NumberGuesser {
    

    public static void main (String [] args){
     

    Scanner inputCollector = new Scanner (System.in);
    
    Random randomNumber = new Random ();
    
    int number = randomNumber.nextInt(100) + 1;
          
   int num = 0;
   
   for (int count = 1; count <= 5;count++) {
    System.out.print("Enter A Number from 1 to 100: ");
    int userInput = inputCollector.nextInt();
    
   if (userInput == number) {
       num += count;
       if (count == 1) {
  System.out.println("The Correct Number is " + userInput + " With "+ count + "attempts used" + "and a Legendary rating");       
       }
     else if (count == 2) {
 System.out.println("The Correct Number is " + userInput + " With "+ count + "attempts used" + "and a Excellent rating");  
     }
        else if (count == 3 || count == 4) {
 System.out.println("The Correct Number is " + userInput + " With "+ count + "attempts used" + "and a Good rating");  
        }
       else if (count == 5) {
 System.out.println("The Correct Number is " + userInput + " With "+ count + "attempts used" + "and a Close rating");  
       }
       
       break;
        }
        
   else if (userInput != number){
   if (userInput < number) {
   System.out.println("Lower than Stake");
   }
   else if (userInput > number) {
   System.out.println("Higher than Stake");
   } 
   
     
   }
   } 
    System.out.println();   
    System.out.println("Ohh! Chances ran out");
    System.out.println("Better Luck Next Time");
   
    
   }



    
    }
    
    
    
    
